runtime/src/governance/election.rs

```diff
@@ -753,8 +757,6 @@ decl_module! {
             }
         }
 
-        // fn withdraw_candidacy()
```

Comment:
Would be good to create issues for this marked as an enhancement:
- Ability to withdraw/cancel a council application
- Ability to withdraw/cancel a vote

Reply:
see #5

Reply:
ah.. right!